src/data/requirements.js

```diff
@@ -4,63 +4,63 @@ export default {
 		2: 'Get 50 kills while ADS',
 		3: 'Get 10 double kills',
 		4: 'Get 10 kills from behind',
-		Gold: null,
+		Gold: 'Get 3 kills without dying 10 times',
 		Polyatomic: null,
 	},
 	'Battle Rifles': {
-		1: null,
+		1: 'test',
```

Comment:
Remove 'test' string 😄 
```suggestion
		1: null,
```